sikhifm_db_back.sql

```diff
@@ -0,0 +1,265 @@
+  `Created` datetime DEFAULT current_timestamp(),
+  `Updated` datetime DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+  PRIMARY KEY (`ID`),
+  KEY `Prefix` (`Prefix`),
+  KEY `Location` (`Location`),
+  FULLTEXT KEY `Name` (`Name`),
+  CONSTRAINT `Artist_ibfk_1` FOREIGN KEY (`Prefix`) REFERENCES `Options` (`Name`),
+  CONSTRAINT `Artist_ibfk_2` FOREIGN KEY (`Location`) REFERENCES `Location` (`ID`)
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+
+DROP TABLE IF EXISTS `sikhifm_db`.`ConversionQueue`;
+CREATE TABLE `sikhifm_db`.`ConversionQueue` (
+  `ID` bigint(21) NOT NULL AUTO_INCREMENT,
+  `Track` int(11) NOT NULL,
+  `Status` tinyint(4) NOT NULL DEFAULT 0,
+  `Created` datetime DEFAULT current_timestamp(),
+  `Updated` datetime DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+  PRIMARY KEY (`ID`),
+  KEY `Track` (`Track`),
+  CONSTRAINT `ConversionQueue_ibfk_1` FOREIGN KEY (`Track`) REFERENCES `Track` (`ID`)
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+
+DROP TABLE IF EXISTS `sikhifm_db`.`Location`;
+CREATE TABLE `sikhifm_db`.`Location` (
+  `ID` int(11) NOT NULL AUTO_INCREMENT,
+  `City` varchar(255) COLLATE utf8_bin DEFAULT NULL,
+  `State` varchar(255) COLLATE utf8_bin DEFAULT NULL,
+  `Country` varchar(255) COLLATE utf8_bin DEFAULT NULL,
+  `Nickname` varchar(255) COLLATE utf8_bin DEFAULT NULL,
+  `NickNameGurmukhi` varchar(512) COLLATE utf8_bin DEFAULT NULL,
+  PRIMARY KEY (`ID`),
+  FULLTEXT KEY `City` (`City`,`State`,`Country`)
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+
+DROP TABLE IF EXISTS `sikhifm_db`.`Options`;
+CREATE TABLE `sikhifm_db`.`Options` (
+  `ID` int(11) NOT NULL AUTO_INCREMENT,
+  `Type` varchar(256) COLLATE utf8_bin NOT NULL,
+  `Name` varchar(256) COLLATE utf8_bin NOT NULL,
+  `Value` varchar(256) COLLATE utf8_bin DEFAULT NULL,
+  PRIMARY KEY (`ID`),
+  UNIQUE KEY `Type` (`Type`,`Name`),
+  KEY `Name` (`Name`),
+  KEY `Type_2` (`Type`)
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+
+DROP TABLE IF EXISTS `sikhifm_db`.`Playlist`;
+CREATE TABLE `sikhifm_db`.`Playlist` (
+  `ID` int(11) NOT NULL AUTO_INCREMENT,
+  `sID` char(6) COLLATE utf8_bin DEFAULT NULL,
+  `UserID` bigint(20) NOT NULL,
+  `Name` varchar(255) COLLATE utf8_bin NOT NULL,
+  `Description` varchar(256) COLLATE utf8_bin DEFAULT NULL,
+  `Current` tinyint(1) NOT NULL DEFAULT 0,
+  `Tracks` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_bin DEFAULT NULL,
+  `Created` datetime DEFAULT current_timestamp(),
+  `Updated` datetime DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+  PRIMARY KEY (`ID`),
+  UNIQUE KEY `UserID` (`UserID`,`Name`),
+  UNIQUE KEY `sID` (`sID`),
+  KEY `UserID_2` (`UserID`,`Current`),
+  KEY `UserID_3` (`UserID`)
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+
+DROP TABLE IF EXISTS `sikhifm_db`.`Track`;
+CREATE TABLE `sikhifm_db`.`Track` (
```

Comment:
feels like we should put sttm shabadids in here too as an array

thoughts @tsingh777 @ManjotS

Reply:
its in a separate table

Reply:
why not in this same table? how would you join shabad_id to this?

Reply:
we can iterate on this